config/locales/en.yml

```diff
@@ -91,7 +91,8 @@ en:
       invite_user_to_school: "Dear %{user_name} \n\n You have been invited to join the claims service for %{school_name}.\n\n Sign in here %{sign_in_url}"
     schools:
       show:
-        heading: Organisations
+        change_organisation: Change organisation
```

Comment:
We had the "change organisation" button removed from the placements version of this page. Simon said it wasn't necessary anymore because of the nav bars. I wonder if maybe he didn't get a chance to update your prototype yet? I assume it would be the same for both services.

Reply:
So I spoke with Simon, you are partially right. We don't need it for support users but for normal users that have multiple schools we do need it. So this PR is still valid :)